seatunnel-core/seatunnel-core-flink/src/main/java/org/apache/seatunnel/core/flink/command/FlinkApiTaskExecuteCommand.java

```diff
@@ -80,9 +80,10 @@ public void execute() throws CommandExecuteException {
                 FlinkEnvironment> execution = new ExecutionFactory<>(executionContext).createExecution()) {
             prepare(executionContext.getEnvironment(), sources, transforms, sinks);
             execution.start(sources, transforms, sinks);
-            close(sources, transforms, sinks);
         } catch (Exception e) {
             throw new CommandExecuteException("Execute Flink task error", e);
+        } finally {
+            close(sources, transforms, sinks);
```

Comment:
It's a bit strange to add it here, and it won't close if an exception occurs above(`showAsciiLogo`)

Reply:
add `showAsciiLogo` into this try block? but as i know, this `showAsciiLogo` will not throw exception.

Reply:
I mean the above lines of code, but anyway, it's a bit strange to put here

Reply:
> I mean the above lines of code, but anyway, it's a bit strange to put here

but `close` in try block, it won't close resources if an exception occurs `execution.start`, so i move it to finally.

Reply:
example

```java
        List<BaseSource<FlinkEnvironment>> sources = executionContext.getSources();
        List<BaseTransform<FlinkEnvironment>> transforms = executionContext.getTransforms();
        List<BaseSink<FlinkEnvironment>> sinks = executionContext.getSinks();
        try {
            ...
        } catch() {
            ...
        } finally {
             close(sources, transforms, sinks);
        }
```